platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingSessionDataStore.java

```diff
@@ -46,19 +46,20 @@
    * session's attributes.
    *
    * @param id the session id
-   * @param sessionAttributes the session's attributes
+   * @param updatedSessionData the session's attributes
    */
-  public void updateSessionAttributes(String id, Map<String, Object> sessionAttributes) {
+  public void updateSessionAttributes(String id, SessionData updatedSessionData) {
     synchronized (sessionDataMap) {
       SessionData sessionData = sessionDataMap.get(id);
 
-      if (sessionData != null && !sessionData.getAllAttributes().equals(sessionAttributes)) {
+      if (sessionData != null && !sessionData.getAllAttributes().equals(sessionData)) {
```

Comment:
✏️  I just noticed that this should be `updatedSessionData.getAllAttributes()`, not `sessionData`.
It's not a big deal, just a minor performance tweak, so I think we can address this in a follow-on PR.